lib/media/drm_engine.js

```diff
@@ -1506,7 +1507,8 @@ shaka.media.DrmEngine = class {
       return;
     }
     shaka.log.debug('Unwrapping PlayReady request.');
-    const dom = new DOMParser().parseFromString(xml, 'application/xml');
+    const dom = shaka.util.XmlUtils.parseXmlString(xml, 'PlayReadyKeyMessage');
```

Comment:
You should add a new entry to `build/conformance.textproto` that bans the use of `DOMParser` outside of the `lib/util/xml_utils.js` file. That way, future third-party contributors will be automatically warned if they use the `DOMParser` directly.

Reply:
Done!